#######################################################################
# This message was automatically generated by the ark_bridge package
#######################################################################
#
# Source message:    .msg
# Creation:          Thu 13 Apr 2017 12:02:49 PM EDT
#
#######################################################################
time                map_load_time
float32             resolution
int64               width
int64               height
Pose                origin
